EosGround/database/db_setup.py

```diff
@@ -0,0 +1,41 @@
+import os
+import psycopg2
+
+from EosGround.config.config import get_config
+
+
+def configure_db():
```

Comment:
Not sure why github isn't rendering the diff but the only changes I made were moving the file, making filepaths cross-platform, adding some print statements, and adding the test_schema setup